static/map.js

```diff
@@ -42,6 +42,7 @@ function pad(num, size) {
 document.getElementById('pokemon-checkbox').checked = getFromStorage("displayPokemons", "true");
 document.getElementById('gyms-checkbox').checked = getFromStorage("displayGyms", "true");
 document.getElementById('coverage-checkbox').checked = getFromStorage("displayCoverage", "true");
+document.getElementById('pokestops-checkbox').checked = getFromStorage("displayPokestops", false);
```

Comment:
I believe this is still incorrect, shouldn't it be the "false" string instead of a boolean? Just basing off from the code above.